torch/_dynamo/variables/torch.py

```diff
@@ -832,6 +832,20 @@ def speculate_branch(branch):
             # TODO: assert that the true/false return values are
             # consistent
             example_value = true_r.as_proxy().node.meta["example_value"]
+        elif self.value.__name__ == "call_delegate":
```

Comment:
if call_delegate is from executorch/exir - I am not sure we want to reference it by this string notion here... where is def call_delegate?